dataverse-webapp/src/main/webapp/editSingleFile.xhtml

```diff
@@ -0,0 +1,381 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:h="http://java.sun.com/jsf/html"
+      xmlns:ui="http://java.sun.com/jsf/facelets"
+      xmlns:p="http://primefaces.org/ui"
+      xmlns:jsf="http://xmlns.jcp.org/jsf"
+      xmlns:f="http://xmlns.jcp.org/jsf/core"
+      xmlns:o="http://omnifaces.org/ui"
+>
+
+<!--@elvariable id="datasetPage" type="edu.harvard.iq.dataverse.DatasetPage"-->
```

Comment:
Are you sure this would work? the datasetpage is Named("DatasetPage"), I don't think referencing it with small letter would work, it will be misleading.

Reply:
This is no longer needed.